recipe/meta.yaml

```diff
@@ -33,11 +33,12 @@ test:
     - pip
 
 about:
-  home: https://github.com/jtpio/jupyterlab-link-share
+  home: https://github.com/jupyterlab-contrib/jupyterlab-link-share
   summary: JupyterLab Extension to share the URL to a running Jupyter Server
   license: BSD-3-Clause
   license_file: LICENSE
 
 extra:
   recipe-maintainers:
     - jtpio
+    - fcollonval
```

Comment:
cc @fcollonval for https://github.com/jupyterlab-contrib/jupyterlab-contrib.github.io/issues/3

Reply:
Thanks @jtpio